stdpopsim/catalog/MyoLuc/genome_data.py

```diff
@@ -0,0 +1,11661 @@
+        "AAPE02069906": {"length": 6662, "synonyms": []},
+        "AAPE02069907": {"length": 6662, "synonyms": []},
+        "AAPE02069908": {"length": 6662, "synonyms": []},
+        "AAPE02069909": {"length": 6662, "synonyms": []},
+        "AAPE02069910": {"length": 6661, "synonyms": []},
+        "AAPE02069911": {"length": 6661, "synonyms": []},
+        "AAPE02069912": {"length": 6660, "synonyms": []},
+        "AAPE02069913": {"length": 6660, "synonyms": []},
+        "AAPE02069914": {"length": 6660, "synonyms": []},
+        "AAPE02069915": {"length": 6659, "synonyms": []},
+        "AAPE02069916": {"length": 6658, "synonyms": []},
+        "AAPE02069917": {"length": 6656, "synonyms": []},
+        "AAPE02069918": {"length": 6656, "synonyms": []},
+        "GL433068": {"length": 6754, "synonyms": []},
+        "AAPE02069921": {"length": 6654, "synonyms": []},
+        "AAPE02069922": {"length": 6654, "synonyms": []},
+        "AAPE02069923": {"length": 6654, "synonyms": []},
+        "AAPE02069924": {"length": 6653, "synonyms": []},
+        "AAPE02069925": {"length": 6652, "synonyms": []},
+        "AAPE02069926": {"length": 6652, "synonyms": []},
+        "AAPE02069927": {"length": 6651, "synonyms": []},
+        "AAPE02069928": {"length": 6650, "synonyms": []},
+        "AAPE02069929": {"length": 6650, "synonyms": []},
+        "AAPE02069930": {"length": 6650, "synonyms": []},
+        "AAPE02069931": {"length": 6649, "synonyms": []},
+        "AAPE02069932": {"length": 6649, "synonyms": []},
+        "AAPE02069933": {"length": 6649, "synonyms": []},
+        "AAPE02069934": {"length": 6646, "synonyms": []},
+        "GL433069": {"length": 6745, "synonyms": []},
+        "AAPE02069937": {"length": 6645, "synonyms": []},
+        "AAPE02069938": {"length": 6645, "synonyms": []},
+        "AAPE02069939": {"length": 6645, "synonyms": []},
+        "AAPE02069940": {"length": 6644, "synonyms": []},
+        "AAPE02069941": {"length": 6643, "synonyms": []},
+        "AAPE02069942": {"length": 6641, "synonyms": []},
+        "AAPE02069943": {"length": 6641, "synonyms": []},
+        "AAPE02069944": {"length": 6640, "synonyms": []},
+        "GL433070": {"length": 6739, "synonyms": []},
+        "AAPE02069947": {"length": 6638, "synonyms": []},
+        "AAPE02069948": {"length": 6638, "synonyms": []},
+        "AAPE02069949": {"length": 6637, "synonyms": []},
+        "AAPE02069950": {"length": 6637, "synonyms": []},
+        "AAPE02069951": {"length": 6636, "synonyms": []},
+        "AAPE02069952": {"length": 6635, "synonyms": []},
+        "AAPE02069953": {"length": 6634, "synonyms": []},
+        "AAPE02069954": {"length": 6634, "synonyms": []},
+        "AAPE02069955": {"length": 6633, "synonyms": []},
+        "AAPE02069956": {"length": 6633, "synonyms": []},
+        "AAPE02069957": {"length": 6632, "synonyms": []},
+        "AAPE02069958": {"length": 6631, "synonyms": []},
+        "AAPE02069959": {"length": 6631, "synonyms": []},
```

Comment:
Huh, weird. Github got to line 8938 and then said: no more syntax highlighting for you!